kirin.proto

```diff
@@ -16,6 +16,16 @@ extend transit_realtime.TripUpdate.StopTimeUpdate {
     optional string stoptime_message = 1000;
 }
 
+enum StopTimeEventStatus {
+  SCHEDULED = 0;
+  SKIPPED = 1;
+  NO_DATA = 2;
+  ADDED = 3;
```

Comment:
"Funny" this `ADDED` disappeared from gtfsrt proto... :eyes: https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/reference.md#enum-schedulerelationship

Reply:
Maybe because it never was in the original gtfs_rt !